desktop/apps/auctions2/components/Layout.js

```diff
@@ -1,27 +1,40 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import { connect } from 'react-redux'
+import AuctionsBlock from 'desktop/apps/auctions2/components/auctions_block/AuctionsBlock'
 
-function Layout ({ isMobile }) {
+function Layout ({ isMobile, auctions }) {
   return (
     <div>
       {isMobile
         ? <div>
             Hello Mobile!
           </div>
         : <div>
-            Hello Desktop!
+          {auctions.live.length ?
+            <AuctionsBlock
+              key='auctions-block-live'
+              auctions={auctions.live}
+              live={true} />
+          : false }
```

Comment:
A cool thing about JSX is if its falsy, it wont render. Can simplify these ternaries to 

```js
{ auctions.live.length &&
  <AuctionsBlock 
    ... 
  />
```